src/encoded/schemas/ivp_a1.json

```diff
@@ -0,0 +1,682 @@
+    }
+  ],
+  "type": "object",
+  "properties": {
+    "schema_version": {
+      "default": "1"
+    },
+    "reason": {
+      "title": "1. Primary reason for coming to ADC:",
+      "type": "string",
+      "enum": [
+        "1 To participate in a research study",
+        "2 To have a clinical evaluation",
+        "3 Other",
+        "4 Both (to participate in a research study and to have a clinical evaluation)",
+        "9 Unknown"
+      ]
+    },
+    "refersc": {
+      "title": "2a. Principal referral source:",
+      "description": "(if answer is 1 or 2, CONTINUE TO QUESTION 2B; otherwise, SKIP TO QUESTION 3.)",
+      "type": "string",
+      "enum": [
+        "1 Self-referral",
+        "2 Non-professional contact (spouse/partner, relative, friend, coworker, etc.)",
+        "3 ADC participant referral",
+        "4 ADC clinician, staff, or investigator referral",
+        "5 Nurse, doctor, or other health care provider",
+        "6 Other research study clinician/staff/investigator (non-ADC; e.g., ADNI, Women's Health Initiative)",
+        "7 Non-ADC media appeal (e.g., Alzheimer's Association)",
+        "8 Other",
+        "9 Unknown"
+      ]
+    },
+    "learned": {
+      "title": "2b.  If the referral source was self-referral or a non-professional contact, how did the referral source learn of the ADC?",
+      "type": "string",
+      "enum": [
+        "1 ADC advertisement (e.g., website, mailing, newspaper ad, community presentation)",
+        "2 News article or TV program mentioning the ADC study",
+        "3 Conference or community event (e.g., community memory walk)",
+        "4 Another organization's media appeal or website (e.g., Alzheimer's Association, clinicaltrials.gov)",
+        "8 Other",
+        "9 Unknown"
+      ]
+    },
+    "prestat": {
+      "title": "3. Presumed disease status at enrollment:",
+      "type": "string",
+      "enum": [
+        "1 Case, patient, or proband",
+        "2 Control or normal",
+        "3 No presumed disease status"
+      ]
+    },
+    "prespart": {
+      "title": "4. Presumed participation:",
+      "type": "string",
+      "enum": [
+        "1 Initial evaluation only",
+        "2 Longitudinal follow-up planned"
+      ]
+    },
+    "source": {
+      "title": "5. ADC enrollment type:",
+      "type": "string",
+      "enum": [
+        "1 Primarily ADC-funded (Clinical Core, Satellite Core, or other ADC Core or project)",
+        "2 Subject is supported primarily by a non-ADC study (e.g., RO1, including non-ADC grants supporting the FTLD Module participation)"
+      ]
+    },
+    "birthmo": {
+      "title": "6a. Subject's month of birth(MM)",
+      "description": "Date can be submitted as MM.",
+      "type": "integer",
+      "minimum": 1,
+      "maximum": 12
+    },
+    "birthyr": {
+      "title": "6b. Subject's year of birth(YYYY)",
+      "description": "Date can be submitted as YYYY.",
+      "type": "integer",
+      "minimum": 1900
+    },
```

Comment:
![image](https://user-images.githubusercontent.com/38848214/83456684-c4292f00-a425-11ea-8efb-e6d774395cd6.png)
How to set limit? Currently can input even five numbers if user input wrong.
If set a pattern, have to set to string.

Reply:
Set a maximum